### ISSUE TITLE
[BUG] Xcparse not installing with Xcode 13.0 Beta 4

### ISSUE THREAD

**Issue by bboyle18** [ChargePoint/xcparse#62]
**Describe the bug**
Xcparse is failing to install when Xcode 13.0 Beta 4 is installed.
```
/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-package-manager/Sources/Basic/Await.swift:18:28: error: value of type '(@escaping (Result<T, ErrorType>) -> Void) -> Void' has no member 'dematerialize'
    return try await(body).dematerialize()
                    ~~~~~~ ^~~~~~~~~~~~~
make: *** [build] Error 1
```

**Desktop (please complete the following information):**
 - OS: MacOS1 1.5.1
 - xcparse Version 2.1.2
 - Swift version 5.5

**Crash Report**
```
==> Tapping chargepoint/xcparse
Cloning into '/usr/local/Homebrew/Library/Taps/chargepoint/homebrew-xcparse'...
remote: Enumerating objects: 77, done.        
remote: Total 77 (delta 0), reused 0 (delta 0), pack-reused 77        
Receiving objects: 100% (77/77), 8.59 KiB | 2.15 MiB/s, done.
Resolving deltas: 100% (33/33), done.
Tapped 1 formula (15 files, 17.9KB).
==> Installing xcparse from chargepoint/xcparse
==> Cloning https://github.com/ChargePoint/xcparse.git
Cloning into '/Users/distiller/Library/Caches/Homebrew/xcparse--git'...
==> Checking out tag 2.1.2
HEAD is now at 61bfe78 Arrays of XCResultValueTypes do not parse correctly (#52)
==> make install prefix=/usr/local/Cellar/xcparse/2.1.2
Last 15 lines from /Users/distiller/Library/Logs/Homebrew/xcparse/01.make:
Computing version for https://github.com/apple/swift-package-manager.git
Computed https://github.com/apple/swift-package-manager.git at 0.5.0 (6.46s)
Computing version for https://github.com/apple/swift-llbuild.git
Computed https://github.com/apple/swift-llbuild.git at 0.2.0 (0.46s)
Creating working copy for https://github.com/apple/swift-package-manager.git
Working copy of https://github.com/apple/swift-package-manager.git resolved at 0.5.0
Creating working copy for https://github.com/apple/swift-llbuild.git
Working copy of https://github.com/apple/swift-llbuild.git resolved at 0.2.0
'llbuild' /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild: warning: ignoring broken symlink /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild/lib/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm/Demangle/include/llvm
'llbuild' /private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild: warning: Invalid Exclude '/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-llbuild/products/llbuildSwift/llbuild.swift': File not found.
[1/2] Compiling clibc libc.c
remark: Incremental compilation has been disabled: it is not compatible with whole module optimizationremark: Incremental compilation has been disabled: it is not compatible with whole module optimization/private/tmp/xcparse-20210803-1217-1ookof9/.build/checkouts/swift-package-manager/Sources/Basic/Await.swift:18:28: error: value of type '(@escaping (Result<T, ErrorType>) -> Void) -> Void' has no member 'dematerialize'
    return try await(body).dematerialize()
                    ~~~~~~ ^~~~~~~~~~~~~
make: *** [build] Error 1
```

**To Reproduce**
Run `brew install chargepoint/xcparse/xcparse` to install xcparse with Xcode 13.0 Beta 4 installed.

**Expected behavior**
Xcparse should install successfully

**Comment by peterbetos** [comment]
Good catch on pointing this out. Good thing Bitrise was able to show the error. Since Xcode 13 is near, this will be needed.

**Comment by abotkin-cpi** [comment]
@bboyle18 Thanks for pointing this out; we haven't switched to an iOS 15 workflow yet and so hadn't gotten to this.

@rsukumar-cpi I'm going to take a look but if it's more involved, may end up assigning to you now that you're full-time.

**Comment by abotkin-cpi** [comment]
This will be addressed by #63. Will release that in the next day or so.

**Comment by abotkin-cpi** [comment]
This is resolved in v2.2.1.
